README.md

```diff
@@ -21,15 +18,39 @@ conceptual documentation on your Drupal site.
  * To submit bug reports and feature suggestions, or to track changes:
    https://www.drupal.org/project/issues/devportal
 
+ * The customization capabilities of the Devportal module can be expanded by the 'Page builder' module. With it, the user can create highly customizable pages with reusable building blocks.
+
+ * You can add the 'FAQ' custom module to your Devportal. Questions can be grouped by topics and these topics can be used as filters on the FAQ listing page.
+
+ * We recommend you to enable the 'Guides' module, which describes the custom functionalities and collects the links for the related core Drupal capabilities. It is a useful tool for content editors.
```

Comment:
This module doesn't provide any documentation by default, it just gives the ability for writers to upload docs. You can refer to the `README.md` file in this submodule: https://github.com/Pronovix/devportal-drupal-module/blob/8.x-2.x/modules/guides/README.md